data/sql/updates/pending_db_world/rev_1636804117557511400.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1636804117557511400');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 467;
+
```

Comment:
this looks to me like a useless line

Reply:
@patou01

Reply:
I mean yeah, I just take whatever keira gives me... 
![image](https://user-images.githubusercontent.com/2592673/142460011-c72dfb27-8c61-4cd4-94eb-05ad1b2ce6cd.png)

We can remove it, it should already have smartAI.